src/v/kafka/server/group_manager.h

```diff
@@ -171,7 +176,8 @@ class group_manager {
 
     // returns the set of registered groups, and an error if one occurred while
     // retrieving the group list (e.g. coordinator_load_in_progress).
-    std::pair<error_code, std::vector<listed_group>> list_groups() const;
+    std::pair<error_code, std::vector<listed_group>>
+    list_groups(const list_groups_filter_data& filter_data) const;
```

Comment:
could pass a default empty filter or use an overlap to avoid updating all call sites